roles/galaxy-config/tasks/get_node_ip.yml

```diff
@@ -14,9 +14,9 @@
 - name: Set web protocol
   set_fact:
     web_protocol: "https"
-  when:
-    - ingress_tls_secret | length
-    - route_tls_secret | length
+  when: >-
+    ingress_tls_secret | length
+    or route_tls_secret | length
```

Comment:
I think we could even remove the `route_tls_secret` condition here because `web_protocol` is only used in the next task which uses `ingress_type|lower == "ingress"` as condition (and even add the same condition to the web_protocol task)

https://github.com/ansible/galaxy-operator/blob/main/roles/galaxy-config/tasks/get_node_ip.yml#L21-L27

The route scenario uses the `web_url` variable instead

https://github.com/ansible/galaxy-operator/blob/main/roles/galaxy-config/tasks/get_node_ip.yml#L51-L57
https://github.com/ansible/galaxy-operator/blob/main/roles/common/tasks/main.yml#L56-L58

Reply:
@dsavineau 
Thanks for reviewing this, and indeed you are absolutely correct. Updated the PR to simplify the conditional. Thanks!